http.go

```diff
@@ -64,6 +64,64 @@ func (k *Kasse) PostLoginPage(res http.ResponseWriter, req *http.Request) {
 	http.Redirect(res, req, redirect, http.StatusFound)
 }
 
+// GetNewUserPage renders the page to create a new user.
+func (k *Kasse) GetNewUserPage(res http.ResponseWriter, req *http.Request) {
```

Comment:
The prefix -Page isn't used elsewhere, I would leave it out (i.e. "GetNewUser").